src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -150,6 +153,23 @@ def get_location_type_with_default_from_resource_group(cli_ctx):
     arg_group=sku_arg_group,
     options_list=['--elastic-pool'])
 
+compute_model_param_type = CLIArgumentType(
+    arg_group=serverless_arg_group,
+    options_list=['--compute-model'],
+    help='The compute model of the database.',
+    arg_type=get_enum_type(ComputeModelType))
+
+auto_pause_delay_param_type = CLIArgumentType(
+    arg_group=serverless_arg_group,
+    options_list=['--auto-pause-delay'],
+    help='Time in minutes after which database is automatically paused. '
```

Comment:
If I don't specify this parameter, which value will be used? That is, what is the default value?

Reply:
like the other commands above, it will be null, and it is the server code's responsibility to handle the case of null. Currently it is working as expected.